lib/install/config/webpacker.yml

```diff
@@ -34,6 +34,7 @@ development:
     host: localhost
     port: 8080
     https: false
+    hot_reloading: false
```

Comment:
@dhh @gauravtiwari,

1. HMR is typically always used, **BUT**, upgrades to React often put it into an "experimental state".
2. Live-reloading is typically a good choice, but per my comment below, I'd have a really hard time getting every one of my team members to switch to the live reloading or just building the files and letting Rails load the files.